cmake/Conan/binaryen.cmake

```diff
@@ -0,0 +1,23 @@
+# TODO: we don't have Conan package for binaryen for now
+# So we add it as an external project.
+set(BINARYEN_ROOT "${CMAKE_BINARY_DIR}/binaryen")
+file(MAKE_DIRECTORY "${BINARYEN_ROOT}")
+
+set(BINARYEN_GITHUB_HOST "github.com" CACHE STRING "Host for binaryen repo")
```

Comment:
XXX_GITHUB_HOST trick was needed for private repos only (to pass SSH keys in gitlab CI). You don't need it for public binaryen repo.

P.S. As soon as other repos become public it can be recursively removed in other places, a ticket can be created for it